.github/workflows/all_solutions.yml

```diff
@@ -202,7 +202,7 @@ jobs:
           if-no-files-found: error
 
       - name: Convert Code Signing Certificate Into File
-        if: ${{ github.event.release }}
+        if: ${{ github.event.release }} || github.event_name == 'workflow_dispatch'
```

Comment:
This doesn't appear to be the same format as our other multiple condition if-statements.  This is an example from our cancel job:
`if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || ( github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v') )`

The workflow ran fine, however so I don't think this is something we need to change, just noting it in case it comes up later.